e2e/workflowTest/complaints/add-new-complaint.spec.js

```diff
@@ -0,0 +1,58 @@
+import LoginComponent from '../../components/login/login.po';
+import Hooks from '../../utilities/hooks';
+import ComplaintsPage from '../../pages/surveillance/complaints/complaints.po';
+import ComplaintsComponent from '../../components/surveillance/complaints/complaints.po';
+
+let hooks;
+let login;
+let page;
+let complaints;
+
+beforeEach(async () => {
+  login = new LoginComponent();
+  hooks = new Hooks();
+  page = new ComplaintsPage();
+  complaints = new ComplaintsComponent();
+  hooks.open('#/surveillance/complaints');
+  await hooks.waitForSpinnerToDisappear();
+});
+
+describe('As ROLE_ACB user', () => {
```

Comment:
The grammar here seems weird

```suggestion
describe('as a ROLE_ACB user', () => {
```
or

```suggestion
describe('as ROLE_ACB', () => {
```